tools/ci/compile.sh

```diff
@@ -54,10 +54,22 @@ if [ $EXIT_CODE != 0 ]; then
     exit $EXIT_CODE
 fi
 
-echo "============ Checking Scaladocs ============"
+echo "============ Checking Javadocs and Scaladocs ============"
+
+# use the same invocation as on buildbot (https://svn.apache.org/repos/infra/infrastructure/buildbot/aegis/buildmaster/master1/projects/flink.conf)
+run_mvn javadoc:aggregate -Paggregate-scaladoc -DadditionalJOption='-Xdoclint:none' \
```

Comment:
same deal as scala, move the output into a file.

Reply:
I'm sorry that you have to point me to all cases of the same class of problems. 
I have no problem with verbose logs, that's why I don't feel the urge to spend time on controlling the logging behavior. I will address this & rebase to the latest master (to see the build passing)

Reply:
For the scala docs, I'm hiding all the warnings / errors and only show them if the build fails. I'm always showing the maven output.

For the Javadocs, I don't know if the errors are send to stderr.
I have the feeling that it is not worth my time producing a javadoc error to understand how the javadocs are printed (stderr vs stdout). I would rather prefer to spend my time testing the new 1.11 features.
In my opinion, it is okay to have 25000 lines of logs for a regular Flink compile.

If the compile stage passes, you usually don't check its output.
if it fails, you will see the error at the bottom of the file + a lot of helpful debug information above (which you would see anyways).

Reply:
shouldn't matter whether it goes into stdout/stderr; you could just pipe everything into a file and dump the whole thing if an error happened.
Basically a simplified version of what you already did for scala, which should at most be a 1 minute fix.

Reply:
I addressed the issue & rebased to latest master.